examples/nima/fault-tolerance/src/main/java/io/helidon/examples/nima/faulttolerance/FtMain.java

```diff
@@ -14,45 +14,37 @@
  * limitations under the License.
  */
 
-package io.helidon.examples.nima.pico;
-
-import java.util.Optional;
+package io.helidon.examples.nima.faulttolerance;
 
 import io.helidon.config.Config;
 import io.helidon.config.ConfigSources;
 import io.helidon.logging.common.LogConfig;
 import io.helidon.pico.api.Bootstrap;
-import io.helidon.pico.api.BootstrapDefault;
 import io.helidon.pico.api.PicoServices;
 
-/**
- * As simple as possible with a fixed port.
- */
-public final class PicoMain {
-    private PicoMain() {
-    }
-
+public class FtMain {
     /**
      * Start the example.
      *
      * @param args ignored
      */
     public static void main(String[] args) {
-        // todo move to a Níma on Pico module
+        // TODO move to a helidon-inject-runtime module (or similar) to set everything up
```

Comment:
This is the purpose of BootStrap. Why do we need more than that?

Reply:
Because there is no method that actually does a bootstrap? Lines 33  to 51 should be hidden from the user of Helidon application.